partner/partner-management-service/pom.xml

```diff
@@ -115,7 +111,7 @@
 				<dependency>
 					<groupId>io.mosip.kernel</groupId>
 					<artifactId>kernel-auth-adapter</artifactId>
-					<version>${project.version}</version>
+					<version>1.2.0.1</version>
```

Comment:
Use 1.2.0.1-java21-SNAPSNOT version. Also, put this version into a properties. Moreover, why is this need here. It should be loaded from artifactory right?

Reply:
Used property as a version. This is profile specific openapi-doc-generate-profile . This is used to generate api json yaml file which is further used in api docs.